test/test.c

```diff
@@ -334,7 +334,6 @@ bool SetAttributes(HCOSE hHandle, const cn_cbor * pAttributes, int which, int ms
 	const cn_cbor * pValue;
 	int keyNew;
 	cn_cbor * pValueNew;
-	bool f = false;
 
```

Comment:
Rather than getting rid of this variable.  Return it and combine the different items together.   Not sure why I removed the assert.  It is possible that doing this change might cause test failures.  If so let me know.

Reply:
@jimsch ok, it seems like the assert does not work

Reply:
@jimsch could you have a look in this?